# Array of 3D detections with header
std_msgs/Header header
Detection3D[] detections
